### ISSUE TITLE
Exception stod: out of range when restarting a simulation

### ISSUE THREAD

**Issue by GerryR** [OPM/opm-simulators#2717]
I am restarting a simulation, and I got this error:
```
libc++abi.dylib: terminating with uncaught exception of type std::out_of_range: stod: out of range
```
Looking inside the restart file, there are a few wrong numbers that may cause a stod exception (for example, `0.60867986139416-308`, it is also missing the `D` before the `-`). These are in the `XGRP` and `XWEL` sections. If I manually fix these numbers, then the simulation works.

The restart file was generated in Ubuntu, with a compiled opm version (latest commit Thu Jun 25 10:53:04 2020).

**Comment by joakim-hove** [comment]
Integrering - that certainly looks like a bug, er should Fox that. But I would guess you could get by by using unformatted files?

**Comment by GerryR** [comment]
That is true, however I am also extracting some quantities (like pressure) from these files, so the formatted output would be better in this case.

**Comment by joakim-hove** [comment]
> That is true, however I am also extracting some quantities (like pressure) from these files, so the formatted output would be better in this case.

OK - opm-common contains code for working with these unformatted files, both C++ code and also Python wrappers. The code is of course mainly written to facilitate the needs of the simulator itself, but it can certainly be used in other separate stand alone applications.

**Comment by GerryR** [comment]
I see, is there a tutorial/documentation on how to use opm-common with python?

**Comment by joakim-hove** [comment]
> I see, is there a tutorial/documentation on how to use opm-common with python?

There is no tutorial unfortunately - but the tests: https://github.com/OPM/opm-common/tree/master/python/tests should illustrate it quite well. Observe that to get the Python goodness you must pass some flags to cmake while configuring
```
cmake .. -DOPM_ENABLE_PYTHON=ON  -DOPM_INSTALL_PYTHON=ON
```
Make an issue in opm-common if you have problems with this.

**Comment by bska** [comment]
@GerryR : Are you still seeing this issue?
